codes/quantum/qudits/qudit_stabilizer.yml

```diff
@@ -8,9 +8,10 @@ physical: qudits
 logical: qudits
 
 name: 'Qudit stabilizer code'
-introduced: '\cite{arXiv:quant-ph/9705052}'
+introduced: '\cite{arXiv:quant-ph/9705052}\cite{arXiv:quant-ph/0005008}'
```

Comment:
Second ref is for Galois qudit stab code. Please remove. That is entry galois_stabilizer. Let me know if you'd like to add details for those codes from Gottesman's book for extra credit.